blueflood-core/src/main/java/com/rackspacecloud/blueflood/service/CoreConfig.java

```diff
@@ -164,7 +165,6 @@
     ROLLUP_ON_READ_REPAIR_SIZE_PER_THREAD( "5" ),
     ROLLUP_ON_READ_TIMEOUT_IN_SECONDS("10"),
 
-    DELAYED_METRICS_MILLIS("300000"),
```

Comment:
This one is only being used for one purpose. Just to determine if a metric is delayed or not and we use it in the tracking code. Since we plan to keep ROLLUP_DELAY_MILLIS the same 10 mins all the time, it makes sense to use that instead. One less config.